harpy/_cli_types.py

```
@@ -0,0 +1,66 @@
+        super().__init__()
+        self.max_entries = max_entries
+
+    def convert(self, value, param, ctx):
+        try:
+            parts = [i.strip() for i in value.split(',')]
+            if len(parts) != self.max_entries:
+                raise ValueError
+            for i in parts:
+                try:
+                    int(i)
+                except:
```

Ruff

Do not use bare `except`

(E722)